Makefile

```diff
@@ -33,7 +33,6 @@ APP = gatekeeper
 SRCS-y := main/main.c
 
 # Functional blocks.
-SRCS-y += catcher/main.c
```

Comment:
This can happen if you've rebased or changed the order of commits since they were originally written. To get around this, I usually do an interactive rebase, choose to edit each patch, and for each one do the following: `git commit --amend --date=now`. Then `git rebase --continue` to go to the next commit, and repeat the process. The commits will be in the correct order when you push them.